brainstormforce/astra
# AST-1894 - Primary submenus not expandable on mobile when toggle button element is not added.

### Description
The submenus for the primary menu does not work if the toggle button is not added to the header for the Off-Canvas menu. 

Here the primary menu is added to the main header (not the off-canvas section).

### Screenshots
Check this screencast to understand it better: https://share.bsf.io/GGuKZz86.

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
1. Switch to mobile/tablet view in the customizer
2. Add the menu in the header and check the submenu if the toggle button is in the header too
3. Remove the toggle button and the submenu does not open.

### Checklist:
